eng/templates/Azure.ServiceTemplate.Template/src/Azure.ServiceTemplate.Template.csproj

```diff
@@ -18,7 +18,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="Azure.Core.Experimental" />
```

Comment:
Could you explain why we need to remove this line?

Reply:
Azure.Core.Experimental is used to for the features which are under-developing.  Now All Azure.Core.Experimental is merged to Azure.Core, we don't need the separated package anymore.